Ombi.Services/Jobs/RecentlyAdded.cs

```diff
@@ -200,16 +200,17 @@ private void Start(NewletterSettings newletterSettings, bool testEmail = false)
                 html = template.LoadTemplate(sb.ToString());
                 Log.Debug("Loaded the template");
             }
-            
 
 
-            Send(newletterSettings, html, plexSettings, testEmail);
+            string escapedHtml = new string(html.Where(c => !char.IsControl(c)).ToArray());
```

Comment:
For some reason when i was running through XHTML strict compliance checks it warned about the control character, browser ignore them but its not fully compliant. I can remove the check if needed but i just added it at the end.